portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +27,12 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Uses fetch() to get this content and give it back to the user.
+ */
+function getHello() {
```

Comment:
This was in PR #3. Submit that PR and then post this one with only the new changes. It might take some time to figure out how to get the right lines in each PR. It's tricky!